CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * Add improvements for `oauth` authentication, and `keycloak` authorization:
   * Support for re-authentication was added, which also enforces access token lifespan on the Kafka client session
   * Permission changes through Keycloak Authorization Services are now detected by Kafka Brokers
+* Add support for `secretPrefix` property for User Operator to prefix all secret names created from KafkaUser resource.
```

Comment:
The 0.20.0 release is already being released. So this will end up in 0.21.0. Can you create a new section for 0.21.0 on top and move it there?